atmos_model.F90

```diff
@@ -2349,6 +2349,266 @@ subroutine assign_importdata(jdat, rc)
             endif
           endif
 
+! get surface snow area fraction: over land (if cpllnd=true and cpllnd2atm=true)
+!------------------------------------------------
+          fldname = 'mean_snow_area_fraction_lnd'
```

Comment:
Can you please update the import field names to ATM to either remove the "mean" or change them to "inst" here and in the cpl/module_cpl_fields. The fd_nems.yaml file will then need to be updated also. This will make these changes consistent with #704

Reply:
@DeniseWorthen I think I need to wait first #704 is in. Right?

Reply:
We hope to get the series of PRs merged early next week which will include #704. This is to support the documentation effort. I think they are independent because they are new, added fields. But I wanted to be sure that when they are committed they're consistent.

Reply:
@DeniseWorthen I think this is already fixed. I am marking as resolved but let me know if you have any other changes.

Reply:
yes, thanks for resolving all the aliases. It will make things simpler in the future.